src/rajAJun20/IntersectionOfArrays.java

```diff
@@ -0,0 +1,51 @@
+package rajAJun20;
+
+public class IntersectionOfArrays {
+	int countTemp = 0;
+	
+	// Common numbers
+	void getIntersectionOfArray(double[] array1, double[] array2) {
+		double[] tempIntersectionArray = new double[(array1.length)];
```

Comment:
Here you have to first check which array has minimum length, then consider this array length to tempIntersectionArray.

Reply:
@RajAJun20 I am agree with Anshu's comment. Good job @AnshuKJun20